semantic_router/encoders/openai.py

```diff
@@ -20,17 +20,19 @@ def __init__(
         self,
         name: Optional[str] = None,
         openai_api_key: Optional[str] = None,
+        openai_org_id: Optional[str] = None,
         score_threshold: float = 0.82,
         dimensions: Union[int, NotGiven] = NotGiven(),
     ):
         if name is None:
             name = os.getenv("OPENAI_MODEL_NAME", "text-embedding-ada-002")
         super().__init__(name=name, score_threshold=score_threshold)
         api_key = openai_api_key or os.getenv("OPENAI_API_KEY")
-        if api_key is None:
+        org_id = openai_org_id or os.getenv("OPENAI_ORGANIZATION")
```

Comment:
I'm not sure we need a new `org_id` var, I think 

`openai_org_id = openai_org_id or os.getenv("OPENAI_ORGANIZATION")`

Should be okay.